CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## Version 2.3.0-beta1
+- Added the ability to correlate http request made between different components represented by different application insights resources. This feeds into the improved [application map experience](http://aka.ms/AiAppMapPreview).
```

Comment:
It's disabled by default. I'll add it when we will enable it in the default configuration